tap_github/comments.json

```diff
@@ -0,0 +1,98 @@
+{
+  "type": ["null", "object"],
```

Comment:
Can we change this to `"type": "object"`? Conventionally the top level schema entity is an explicit object rather than a candidate for a `null` value.